README.md

```diff
@@ -30,6 +30,26 @@ All updates to this library is documented in our [CHANGELOG](https://github.com/
 
 - Ruby version 2.2+
 
+## Setup Environment Variables
+
+### Initial Setup
+
+Copy ```.env_sample``` and change its name to ```.env.```
```

Comment:
@thepriefy please make the same change here that you made on the other repos - remove one of the sections here.